### PR TITLE
ansible: add httplib2

### PR DESCRIPTION
**Pull request by richardlau**
V8's `gclient.py` now requires `httplib2`. Install it on all the
machines that the V8 CI can run on.

Fixes: https://github.com/nodejs/build/issues/2880

### CODE REVIEW

**Comment by richardlau** [comment]
This proved to be very tricky on the RHEL 7 s390x (LinuxONE) instances. `httplib2` refused to install via `pip2` with the following error:

```console
$ sudo pip2 install httplib2
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Collecting httplib2
  Using cached httplib2-0.20.4.tar.gz (349 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /bin/python /usr/lib/python2.7/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpbgjji9
       cwd: /tmp/pip-install-JBiOqF/httplib2
  Complete output (4 lines):
  Traceback (most recent call last):
    File "/usr/lib/python2.7/site-packages/pip/_vendor/pep517/_in_process.py", line 15, in <module>
      from glob import glob
  ImportError: No module named glob
  ----------------------------------------
ERROR: Command errored out with exit status 1: /bin/python /usr/lib/python2.7/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpbgjji9 Check the logs for full command output.
```

I even tried `sudo pip2 install glob2` (which some searches suggested to install for `glob`) but still got the above error. In the end I had to disable PEP 517 (`--no-use-pep517`) but I had to pick an older version of `httplib2` because the later ones require PEP 517.

And to cap everything off the `adoptopenjdk` tasks were failing due to a mismatched checksum on the downloaded binary compared to the metadata from the AdoptOpenJDK REST API. I've switched the RHEL7 s390x instances back to the `java-1.8.0-openjdk` yum package. I also registered the RHEL 7 instances with our RHEL subscription so that we get the most up to date packages.

File: ansible/roles/baselayout/tasks/main.yml [hunk 1]
```diff
@@ -141,16 +141,6 @@
...
-# Required for V8 builds
-- name: rhel8 | update python package alternatives
-  community.general.alternatives:
-    link: /usr/bin/python
-    name: python
-    path: /usr/bin/python2
-  when:
-    - os == "rhel8"
-    - build_test_v8|default(False)
-
```
Comment (richardlau) [inline ansible/roles/baselayout/tasks/main.yml:153]: Moved to `ansible/roles/build-test-v8/tasks/partials/rhel8.yml`.

File: ansible/roles/baselayout/vars/main.yml [hunk 2]
```diff
@@ -144,10 +144,6 @@ packages: {
     'gcc-c++,sudo,git,zip,unzip,iptables-services,GConf2-devel,openssl-devel,python3',
   ],
 
-  rhel8_s390x: [
-    'GConf2-devel,python2' # Needed for V8 builds
-  ],
-
```
Comment (richardlau) [inline ansible/roles/baselayout/vars/main.yml:150]: Moved to `ansible/roles/build-test-v8/tasks/partials/rhel8.yml`.

**Comment by targos** [comment]
> This proved to be very tricky on the RHEL 7 s390x (LinuxONE) instances. httplib2 refused to install via pip2

Maybe the tools are now able to run with Python 3?

**Comment by richardlau** [comment]
> > This proved to be very tricky on the RHEL 7 s390x (LinuxONE) instances. httplib2 refused to install via pip2
> 
> Maybe the tools are now able to run with Python 3?

We'd still need to be able to run the job against older Node.js release lines.

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**